src/TLuaInterpreter.cpp

```diff
@@ -7989,6 +8033,71 @@ int TLuaInterpreter::isActive(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#isActiveID
+int TLuaInterpreter::isActiveID(lua_State* L)
+{
+    auto id = getVerifiedInt(L, __func__, 1, "item ID");
+    if (id < 0) {
+        // Must be zero or more but doesn't seem to be:
+        return warnArgumentValue(L, __func__, "item ID as %d does not seem to be a positive integer", id);
+    }
+
+    // Although we only use 4 ASCII strings the user may not enter a purely
+    // ASCII value which we might have to report...
+    QString type = getVerifiedString(L, __func__, 2, "item type");
+
+    Host& host = getHostFromLua(L);
+    if (!type.compare(QLatin1String("timer"), Qt::CaseInsensitive)) {
+        auto pT = host.getTimerUnit()->getTimer(id);
+        if (!pT) {
+            return warnArgumentValue(L, __func__, "timer ID %d does not exist", id);
+        }
+        lua_pushboolean(L, pT->isActive());
+        return 1;
+    }
+    if (!type.compare(QLatin1String("trigger"), Qt::CaseInsensitive)) {
+        auto pT = host.getTriggerUnit()->getTrigger(id);
+        if (!pT) {
+            return warnArgumentValue(L, __func__, "trigger ID %d does not exist", id);
+        }
+        lua_pushboolean(L, pT->isActive());
+        return 1;
+    }
+    if (!type.compare(QLatin1String("alias"), Qt::CaseInsensitive)) {
+        auto pT = host.getAliasUnit()->getAlias(id);
+        if (!pT) {
+            return warnArgumentValue(L, __func__, "alias ID %d does not exist", id);
+        }
+        lua_pushboolean(L, pT->isActive());
+        return 1;
+    }
+    if (!type.compare(QLatin1String("keybind"), Qt::CaseInsensitive)) {
+        auto pT = host.getKeyUnit()->getKey(id);
+        if (!pT) {
+            return warnArgumentValue(L, __func__, "key-binding ID %d does not exist", id);
+        }
+        lua_pushboolean(L, pT->isActive());
+        return 1;
+    }
+    if (!type.compare(QLatin1String("button"), Qt::CaseInsensitive)) {
+        auto pT = host.getActionUnit()->getAction(id);
+        if (!pT) {
+            return warnArgumentValue(L, __func__, "button/menu/toolbar ID %d does not exist", id);
```

Comment:
The player named the type as "button" so let's not respond much more complicated than that.
```suggestion
    if (!type.compare(QLatin1String("button"), Qt::CaseInsensitive)) {
        auto pT = host.getActionUnit()->getAction(id);
        if (!pT) {
            return warnArgumentValue(L, __func__, "button (or menu/toolbar) ID %d does not exist", id);
```

Reply:
Would `"button (or menu or toolbar) ID %d does not exist"` be acceptable - either we use `/` between them or ` or ` - since the original (and shortest) using just `/` doesn't appeal to you?